index.js

```diff
@@ -41,3 +41,77 @@ app.get('/dogs', (req, res) => {
         })
     })
 })
+
+app.post('/dogs/:id/win', jsonParser, (req, res) => {
```

Comment:
(my fault, apologies) but I _think_ when we're creating things, we should have plurals, so `/dogs/:id/wins`

Reply:
(alternatively, thinking about the next story we've pulled in) now might be a good time to argue for ` POST /result` as we will be creating a result that isn't tied to a single dog. Thoughts?

Reply:
on it

Reply:
done